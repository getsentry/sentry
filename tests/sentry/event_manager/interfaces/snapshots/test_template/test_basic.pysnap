---
created: '2019-03-14T17:12:36.653397Z'
creator: sentry
source: tests/sentry/event_manager/interfaces/test_template.py
---
api_context:
  context:
  - - 1
    - hello world
  filename: foo.html
  lineNo: 1
errors: null
to_json:
  context_line: hello world
  filename: foo.html
  lineno: 1
to_string: 'Stacktrace (most recent call last):




  File "foo.html", line 1


  hello world'
