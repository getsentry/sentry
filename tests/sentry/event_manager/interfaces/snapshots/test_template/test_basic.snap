api_context:
  context:
  - [1, hello world]
  filename: foo.html
  lineNo: 1
errors: null
to_json: {context_line: hello world, filename: foo.html, lineno: 1}
to_string: 'Stacktrace (most recent call last):




  File "foo.html", line 1


  hello world'
